### ISSUE TITLE
Integrate flask-webfonts extension with SILPA framework

### ISSUE THREAD

**Issue by copyninja** [libindic/Silpa-Flask#17]
As part of Gsoc 2013 we created [flask-webfonts](https://github.com/Project-SILPA/flask-webfont) extension but this is not integrated yet and needs to be integrated so people who are already utilizing webfonts from SILPA can continue to do so.

This requires tuning flask-webfonts and creating tool which can take directories of font and create required config file which can be used by extension

**Comment by aashay201297** [comment]
The link to flask-webfonts is broken. Can you please elaborate on this issue?

**Comment by asdofindia** [comment]
@aashay201297, good catch on the broken link. Here's the correct link: [flask-webfonts](https://github.com/libindic/flask-webfonts

**Comment by aashay201297** [comment]
@asdofindia thanks, i am starting work on this issue. If there is any work already done, please let me know so that i can work optimally.
